index.bs

```diff
@@ -573,12 +573,12 @@ run these steps:
 <h4 id='calculate-intersection-rect-algo'>
 Compute the Intersection of a Target Element and the Root</h4>
 
-To <dfn>compute the intersection</dfn> between a <a>target</a> and the observer's <a>intersection root</a>,
+To <dfn>compute the intersection</dfn> between an input <a>target</a> |target| and an input |root| <a>intersection root</a>,
```

Comment:
Can the types of the inputs be stated more clearly? Preferably using `a <type> |name|` syntax.

E.g. |target| must be an element since you're running getBoundingClientRect() on it, right? What about root?

Reply:
Done.